CONTRIBUTING.md

```diff
@@ -56,8 +56,9 @@ cannot be merged.
 Do:
 * enable strings visible in the Mudlet GUI to be translateable
 * minimise use of HTML styling tags in strings to be translated
+* enable users to use language-specific Mudlet object names (triggers, aliases, labels, etc)
 
 Don't:
-* translate the Mudlet API: functions, events, or error messages
+* translate the Mudlet API: functions, events, error messages or constants (`main` console, mapper directions)
```

Comment:
OK. I will take that part out for now. I agree they need to be customisable, but in a way that is transparent to the mapper script.